python/tvm/__init__.py

```diff
@@ -63,9 +63,11 @@
 # Contrib initializers
 from .contrib import rocm as _rocm, nvcc as _nvcc, sdaccel as _sdaccel
 
+PREV_EXCEPTHOOK = sys.excepthook
+
 # Clean subprocesses when TVM is interrupted
 def tvm_excepthook(exctype, value, trbk):
-    print('\n'.join(traceback.format_exception(exctype, value, trbk)))
```

Comment:
Let us modify the function as `tvm_wrap_excepthook(exception_hook)` that returns a new wrapped hook, and then do

sys.excepthook = tvm_wrap_exceptionhook(sys.except_hook)